src/userbehavior/userbehavior/usbing/usbing.py

```diff
@@ -121,8 +121,11 @@ def take_timeout(self, secs=None):
     def save_exes_from_usb_device(self):
         logger.info("Saving exes on USB device to tmp folder")
         self.mount_usb_device()
-        self.save_exes_from_mounted_device()
-        self.unmount_usb_device()
+        if self.usb_device.is_mounted():
```

Comment:
This works, but slightly breaks the coding style of this file. This function should avoid using usb_device directly and instead use a wrapper function like `self.is_mounted`.